docs/EiffelVis/content/_index/features-2/hovering.md

```diff
@@ -0,0 +1,17 @@
++++
+fragment = "item"
+#disabled = false
+date = "2022-01-05"
+weight = 115
+background = "dark"
+align = "top"
+
+title = "Node Hover"
```

Comment:
We may have talked about it before, but I suggest using sentences like "Easily highlight events" rather than "Node Hover"; it sounds way more interesting and increases the chance that people will actually read whatever is below